reana_commons/config.py

```diff
@@ -350,14 +356,12 @@ def workspaces(paths):
 WORKSPACE_PATHS = workspaces(json.loads(os.getenv("WORKSPACE_PATHS", "{}")))
 """Dictionary of available workspace paths with pairs of cluster_node_path:cluster_pod_mountpath."""
 
+AVAILABLE_WORKSPACE_PATHS = list(WORKSPACE_PATHS.values())
```

Comment:
I think it may be confusing to have both `WORKSPACE_PATHS` and `AVAILABLE_WORKSPACE_PATHS` as two different configuration variables for denoting the same thing.

Perhaps you could keep only `WORKSPACE_PATHS` and do the `list()` in each use case occurrence, or else introduce a convenience function `get_available_workspace_paths()` that would return list that could be directly consumed, and `get_default_workspace()` to return default one, etc. Then one-config, several-convenience-functions might result in more readable code.

P.S. That said, when I see `WORKSPACE_PATHS`, my first reaction is to think about a list, not a dictionary... So we may have a slight misnomer there too.

Reply:
Indeed, reverted to the previous logic where we had only `WORKSPACE_PATHS` and used `list()` in each of the use case as suggested